lib/stream.mjs

```diff
@@ -8,7 +8,7 @@ export default class Stream {
     }
 
     // Get the element at the cursor.
-    head() {
+    head(len=1) {
```

Comment:
Nit: JS style is to use spaces around the `=` here.

Reply:
Adding a config to eslint to tell me next time.